main.go

```diff
@@ -18,15 +18,18 @@ import (
 	"go.uber.org/zap"
 )
 
-const appName = "new-relic-k8s-metadata-injection"
+const (
+	appName        = "new-relic-k8s-metadata-injection"
+	defaultTimeout = time.Second * 30
+)
 
 // Specification contains the specs for this app.
 type Specification struct {
 	Port        int           `default:"443"`                                                      // Webhook server port.
 	TLSCertFile string        `default:"/etc/tls-key-cert-pair/tls.crt" envconfig:"tls_cert_file"` // File containing the x509 Certificate for HTTPS.
 	TLSKeyFile  string        `default:"/etc/tls-key-cert-pair/tls.key" envconfig:"tls_key_file"`  // File containing the x509 private key for TLSCERTFILE.
 	ClusterName string        `default:"cluster" split_words:"true"`                               // The name of the Kubernetes cluster.
-	Timeout     time.Duration // server timeout. Defaults to the timeout passed by K8s API via query param.
+	Timeout     time.Duration // server timeout. Defaults to the timeout passed by K8s API via query param. If not present, to the defaultTimeout const value.
```

Comment:
is not possible to use the default tag here?

Reply:
My first idea was to let the timeout behave like:

1. If the user sets any value, use it.
2. If the user *does not* set any value:
	- If K8s calls the webhook with the `?timeout=` query param, use it.
    - If there is no `?timeout=` query param, use the default value (const).

Reply:
Ah, ok, that makes sense. Thank you for the explanation.